### ISSUE TITLE
Renewals portal other tasks

### ISSUE THREAD

**Issue by jrosindell** [OneZoom/OZtree#361]
We will need to phrase e-mail reminders to people

1. reminder that you need to review (with / without having received a free extension to the timeframe)
2. second reminder that you need to review (with / without having received a free extension to the timeframe)
3. thanks for renewing email

We need to decide on a pricing strategy

what if the recommended donation for a species has changed due to popularity updates 
should we offer a discount for the renewal e.g. 20% off to encourage renewal, but only if price has not decreased

**Comment by jrosindell** [comment]
We also need to decide about renewals of leaves that were originally sponsored through partners.

**Comment by jrosindell** [comment]
new price = current price *0.8 regardless of circumstances

**Comment by jrosindell** [comment]
sponsorship_text_level will be used to determine who gets e-mailed automatically and who gets a personalised e-mail
sponsorship_text_level  = NULL should mean they get e-mailed as normal.

**Comment by jrosindell** [comment]
we will need to handle the 'contact us' price cases differently

**Comment by jrosindell** [comment]
@hyanwong here is what I think we've decided....

new price = current price *0.8 regardless of circumstances
partners get the same percentage as before.
The only thing I'm not sure about is how we're going to handle sponsorship_text_level - this is a management decision that we need to make soon.

**Comment by lentinj** [comment]
> new price = current price *0.8 regardless of circumstances

This is controlled via. ``sponsorship.renew_discount``, the default being 0.2

> partners get the same percentage as before.

@jrosindell "before" meaning we should copy ``partner_percentage`` from the previous sponsorship, or we should find the partner data and copy it from there? The latter is the right choice IMO, if that helps.

- [ ] Populate ``partner_percentage`` in ``reservation_confirm_payment``

> how we're going to handle sponsorship_text_level

Again, we're not copying that over as yet.

> we need to check what the renewals portal does [for 'contact us' price cases]

They're listed in the expiry table like any other OTT, but you can't buy them. Email reminders don't exist yet, but I don't think this needs to be a special case.

**Comment by hyanwong** [comment]
> > new price = current price *0.8 regardless of circumstances
> 
> This is controlled via. `sponsorship.renew_discount`, the default being 0.2

Ah, I guess I'll need to add that to the server setup.

**Comment by lentinj** [comment]
> > > new price = current price *0.8 regardless of circumstances
> > 
> > 
> > This is controlled via. `sponsorship.renew_discount`, the default being 0.2
> 
> Ah, I guess I'll need to add that to the server setup.

You don't have to, the default is correct (when applied it's ``price * (1 - renew_discount)``).

**Comment by jrosindell** [comment]
> partners get the same percentage as before.
@jrosindell "before" meaning we should copy partner_percentage from the previous sponsorship, or we should find the partner data and copy it from there? The latter is the right choice IMO, if that helps.

Agreed - we should collect the partner_percentage from the partner data table rather than copy across

**Comment by jrosindell** [comment]
> > we need to check what the renewals portal does [for 'contact us' price cases]
> They're listed in the expiry table like any other OTT, but you can't buy them. Email reminders don't exist yet, but I don't think this needs to be a special case.

If 'banned' species with no renewal price come up on the renewal page then the user should just get a bit of text saying "This species is a special case. You can still renew your sponsorship but you will need to contact us."

The e-mail reminder system should not send e-mails to the user about these species, it should instead send the same e-mail text to mail@onezoom.org for admin handling.

**Comment by lentinj** [comment]
> If 'banned' species with no renewal price come up on the renewal page then the user should just get a bit of text saying "This species is a special case. You can still renew your sponsorship but you will need to contact us."

Exactly. If you can't renew, since e.g. it's banned, you see "To renew, please write to us".

> The e-mail reminder system should not send e-mails to the user about these species, it should instead send the same e-mail text to mail@onezoom.org for admin handling.

Added to #439.

**Comment by lentinj** [comment]
partner_percentage is now re-loaded and updated on renew or expire-and-repurchase. I don't think there's any other actions for me or Yan here now.
